practitioner-service/src/main/resources/bootstrap-dev.properties

```diff
@@ -1,6 +1,6 @@
 spring.application.name=practitioner-service
```

Comment:
Should common values for all profiles be unified under bootstrap.properties?

Reply:
renamed to bootstrap.properties
verified it works